src/memex/storage.py

```diff
@@ -108,14 +109,14 @@ def create_annotation(request, data):
             )
 
     # The user must have permission to create an annotation in the group
-    # they've asked to create one in.
-    if data['groupid'] != '__world__':
-        group_principal = 'group:{}'.format(data['groupid'])
-        if group_principal not in request.effective_principals:
-            raise schemas.ValidationError('group: ' +
-                                          _('You may not create annotations '
-                                            'in groups you are not a member '
-                                            'of!'))
+    # they've asked to create one in. If the application didn't configure
+    # a groupfinder we will allow writing this annotation without any
+    # further checks.
+    group = groups.find(request, data['groupid'])
```

Comment:
The groupfinder specifically returns `None`, and `has_permission` on `None` will return `False`. So an API client will get a 400 error: `{"status": "failure", "reason": "group: You may not create annotations in the specified group!"}`

That's the correct behaviour in my opinion, no?